### DISCUSSION TITLE
`CSSObject` type

### DISCUSSION THREAD

**Discussion by Semigradsky** [garronej/tss-react#76] [General]
https://github.com/garronej/tss-react/blob/main/src/tools/types/CSSObject.ts

```ts
/** 
 * Copy of { CSSObject } from "@emotion/react" 
 * We copy it instead of importing it directly because 
 * CSSInterpolation is not exposed in "@emotion/react"
 * */
 ```
 
 As I see all these types can be imported from `"@emotion/serialize"`.

**Comment by garronej** [comment]
Cool!  
I see it's even exported from `@emotion/react` now: https://github.com/emotion-js/emotion/blob/04681a5f520b3e3909e567fda200622494dd333b/packages/react/types/index.d.ts#L30  

Thank you for reporting
